2018-11-16T11:45:40|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\Users\Admin\Desktop\Camera-ScopeFoundry\CameraApp.py", line 26, in <module>
    app = HamamatsuApp(sys.argv)
  File "C:\ProgramData\Anaconda3\lib\site-packages\ScopeFoundry\base_app.py", line 332, in __init__
    self.setup()
  File "C:\Users\Admin\Desktop\Camera-ScopeFoundry\CameraApp.py", line 10, in setup
    self.add_hardware(HamamatsuHardware(self))
  File "C:\ProgramData\Anaconda3\lib\site-packages\ScopeFoundry\hardware.py", line 80, in __init__
    self.setup()
  File "C:\Users\Admin\Desktop\Camera-ScopeFoundry\CameraHardware.py", line 57, in setup
    self.optimal_offset
AttributeError: 'HamamatsuHardware' object has no attribute 'optimal_offset'

